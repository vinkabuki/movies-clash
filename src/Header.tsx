import React from 'react'
import styled from 'styled-components'

import logo from './assets/logo.jpg'

const LogoImg = styled.img`
  width: 30px;
  height: 30px;
`

const Header: React.FC = () => {
  return (
    <div>
      <h2>
        Movies Clash
        <span>
          <LogoImg src={logo} alt="Our logo" />
        </span>
      </h2>
    </div>
  )
}

export default Header
