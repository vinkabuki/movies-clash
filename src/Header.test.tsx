import React from 'react'
import renderer from 'react-test-renderer'
import Header from './Header'

describe('Header component basic test', () => {
  test('Render header component', () => {
    const component = renderer.create(<Header />)
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})
