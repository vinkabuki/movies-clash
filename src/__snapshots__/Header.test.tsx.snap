// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component basic test Render header component 1`] = `
<div>
  <h2>
    Movies Clash
    <span>
      <img
        alt="Our logo"
        className="sc-AxjAm dYgbqu"
        src="logo.jpg"
      />
    </span>
  </h2>
</div>
`;
